### PR TITLE
Add tooltips for sections

### PR DESCRIPTION
**Pull request by AdityaKhatri**

### CODE REVIEW

File: src/newViews/AnalyticalFramework/PrimaryTagging/SectionsEditor/index.tsx [hunk 1]
```diff
@@ -124,6 +152,7 @@ interface Props {
     onCancel: () => void;
     onSave: (value: Section[]) => void;
     onChange: (value: PartialSectionType[]) => void;
+    sectionToFocus?: string;
```
Comment (tnagorra) [inline src/newViews/AnalyticalFramework/PrimaryTagging/SectionsEditor/index.tsx]:
```suggestion
    focusedSection?: string;
```

File: src/newViews/AnalyticalFramework/PrimaryTagging/index.tsx [hunk 2]
```diff
@@ -317,6 +320,18 @@ function PrimaryTagging(props: Props) {
         [sections, tempSections, tempWidget],
     );
 
+    const [sectionToEdit, setSectionToEdit] = useState<string | undefined>(undefined);
+
+    const handleSectionEditClick: ButtonProps<string>['onClick'] = useCallback((newSectionToEdit, event) => {
```
Comment (tnagorra) [inline src/newViews/AnalyticalFramework/PrimaryTagging/index.tsx:325]: Please write types

File: src/newViews/AnalyticalFramework/PrimaryTagging/SectionsEditor/index.tsx [hunk 3]
```diff
@@ -89,33 +96,54 @@ function SectionInput(props: SectionInputProps) {
...
+                    error={error?.fields?.title}
+                    autoFocus={autoFocus}
+                />
+            )}
+            className={_cs(
+                className,
+                autoFocus && styles.focus,
+            )}
+            footerContent={(
+                <NonFieldError error={error} />
```
Comment (tnagorra) [inline src/newViews/AnalyticalFramework/PrimaryTagging/SectionsEditor/index.tsx]: Why move this here?

File: src/newViews/AnalyticalFramework/PrimaryTagging/SectionsEditor/index.tsx [hunk 4]
```diff
@@ -188,54 +218,61 @@ function SectionsEditor(props: Props) {
...
+                            disabled={pristine}
+                            // FIXME: use strings
+                        >
+                            Save
+                        </Button>
+                    </>
+                )}
+                sub
+            >
+                {value.sections?.map((section, index) => (
```
Comment (tnagorra) [inline src/newViews/AnalyticalFramework/PrimaryTagging/SectionsEditor/index.tsx:288]: NonFieldError for sections is missing

File: src/newViews/AnalyticalFramework/PrimaryTagging/index.tsx (hunk 2 above)
Comment (AdityaKhatri) [reply src/newViews/AnalyticalFramework/PrimaryTagging/index.tsx:325]: sir eyes where?

**Review by tnagorra — APPROVED** [review]

**Pull request merged**